src/test/java/dl/news/portal/web/controller/NewsControllerTest.java

```diff
@@ -0,0 +1,125 @@
+package dl.news.portal.web.controller;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import dl.news.portal.domain.dto.NewsDto;
+import dl.news.portal.domain.entity.News;
+import dl.news.portal.domain.entity.User;
+import dl.news.portal.domain.service.NewsService;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.util.Optional;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(SpringRunner.class)
+@WebMvcTest(NewsController.class)
+public class NewsControllerTest {
+    private static final String NEWS_PATH = "/news";
+    @Autowired
+    private MockMvc mockMvc;
+    @MockBean
+    private NewsService newsService;
+
+    @Test
+    public void getNewsById_whenNewsExists_shouldReturnOk() throws Exception {
+        final String path = NEWS_PATH + "/1";
+        User user = new User();
+        user.setId(2L);
+        News news = new News();
+        news.setId(1L);
+        news.setContent("test");
+        news.setTitle("test");
+        news.setAuthor(user);
+
+        Mockito.when(newsService.findNewsById(1L)).thenReturn(Optional.of(news));
+        mockMvc.perform(get(path)).andExpect(status().isOk());
```

Comment:
Add what else test expect for example fields

Reply:
Done